config/initializers/git.rb

```diff
@@ -0,0 +1,3 @@
+Git.configure do |config|
+  config.git_ssh = ENV['bCHATTO_HUB_ADMIN_GIT_SSH']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.